source/RialtoGStreamerMSEBaseSink.cpp

```diff
@@ -132,10 +132,6 @@ static void rialto_mse_base_sink_error_handler(RialtoMSEBaseSink *sink, firebolt
 static void rialto_mse_base_sink_rialto_state_changed_handler(RialtoMSEBaseSink *sink,
                                                               firebolt::rialto::PlaybackState state)
 {
-    // Wait for lost state to complete before handling a state notification,
-    // otherwise states can change while we are processing.
-    std::unique_lock<std::mutex> lock(sink->priv->m_lostStateMutex);
-
     sink->priv->m_serverPlaybackState = state;
```

Comment:
We can remove this variable all together now that the logic has move to GStreamerMSEMediaPlayerClient